packages/config/config/devices/0x000c/templates/homeseer_template.json

```diff
@@ -105,6 +105,39 @@
 		"$import": "~/templates/master_template.json#dimming_timing",
 		"valueSize": 2
 	},
+	"dimming_threshold": {
+		"label": "Dimming Threshold",
+		"description": "Sets the lowest dimming threshold",
+		"valueSize": 1,
+		"minValue": 1,
+		"maxValue": 14,
+		"defaultValue": 1
+	},
+	"ramp_rate": {
+		"valueSize": 1,
+		"unit": "seconds",
+		"minValue": 0,
+		"maxValue": 90,
+		"defaultValue": 3
+	},
+	"scene_control": {
+		"label": "Central Scene, Multi-Tap, Press and Hold",
```

Comment:
Let's just make this label "Central Scene Commands"

Reply:
Updated.